Input/InputField.js

```diff
@@ -291,13 +292,13 @@ const InputFieldBase = kind({
 				<input
 					{...inputProps}
 					{...voiceProps}
-					className={css.input}
+					className={type === 'passwordtel' ? classnames(css.input, css.passwordtel) : css.input}
```

Comment:
Why don't we use `computed` block and define `inputClassName`?

Reply:
inputClassName: ({styler, type}) => styler.append('input', {passwordtel: (type === 'passwordtel')}),

Reply:
Fix it